CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+## Welcome
+
+If you are a student at North Seattle College we welcome you to contribute to our project! Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**. If you are nervous that is okay. The community will help guide you. Mistakes will be made, we will all learn together, and everything will be fine!
+
+Not all contributions are in the form of code. In fact, a great way to learn about the project is dig into the documentation and poke around the website. Then do any of the following:
+
+- Add new or editing existing documentation
+- Submit a bug report
+- Submit a feature request
+- Ask a question
+
+## How to Contribute
+Navigate to the `Issues` tab and use the filters to see open issues. Take note of the `good first issue` and `help wanted` labels.
+
+#### Submit a Bug
+- Navigate to `Issues` and click the `New Issue` button
+- Select `Bug Report`
+
+#### Submit a Feature Request
+- Navigate to `Issues` and click the `New Issue` button
+- Select `Feature Request`
+
+#### Prerequisites
+
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`.
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`.
+- Install Node.js
+
+##### Windows
+- Install current version of Python from Microsoft Store.
+- Install Visual Studio Code.
+    - Add the Microsoft C/C++ extention.
+- Install Node.js for Windows.
+- OR Install Windows Subsystem for Linux 2 and follow the linux instructions (not recommended unless familiar with WSL already) 
+
+##### Linux
+
+- Install Python > 3.7
+- Install 'make'
+- A C/C++ compiler toolchain (the GNU Compiler Collection for example)
+
+##### Then
+
+- OPen a terminal then fork this project to your projects folder.
```

Comment:
Typo, and you probably mean clone instead of fork. There should be a step to fork above, then they should clone that fork. Also, `projects` folder might need clarification or be changed to Documents or something.